docs/man/rpm-plugin-unshare.8.md

```diff
@@ -27,6 +27,10 @@ This plugin implements the following configurables:
     execution. Typical examples would be `/tmp` to protect against
     insecure temporary file usage inside scriptlets, and `/home` to
     prevent scriptlets from accessing user home directories.
+    When path unsharing is enabled, any mounts made from scriptlets
+    are also private to the scriptlet.
```

Comment:
This should also mention that any mounts made outside the namespace (i.e. on the host) are invisible to the namespace as well.

Reply:
Although we're mostly interested in the former (mounts done by scriptlets leaking outside) so the latter only needs a brief mention.

Reply:
Mentioning doesn't hurt either - added.